sdgym/synthesizers/generate.py

```diff
@@ -147,7 +148,7 @@ def create_multi_table_synthesizer(get_trained_synthesizer_fn, sample_from_synth
         class:
             The synthesizer class.
     """
-    class NewSynthesizer(BaselineSynthesizer):
+    class NewSynthesizer(MultiSingleTableBaselineSynthesizer):
```

Comment:
just a question but why is it called `MultiSingleTable` instead of just `MultiTable`?

Reply:
We use `MultiSingleTable` in SDGym and SDMetrics. It already existed in the repos so I was just maintaining consistency, but we can discuss the naming further offline.